api/v1alpha1/proxydefaults_types.go

```diff
@@ -176,6 +176,11 @@ func (in *ProxyDefaults) Validate(namespacesEnabled bool) error {
 	return nil
 }
 
+// Default has no behaviour here as proxy-defaults have no namespace specific fields.
```

Comment:
```suggestion
// DefaultNamespaceFields has no behaviour here as proxy-defaults have no namespace specific fields.
```

Also the other locations.